jest.project.js

```diff
@@ -1,5 +1,5 @@
 const { resolve } = require('path');
-const { pathsToModuleNameMapper } = require('ts-jest/utils');
```

Comment:
getting `ts-jest[main] (WARN) Replace any occurrences of "ts-jest/utils" with just "ts-jest".` when running jest if not